celavi/pylca_celavi/pylca_opt_background.py

```diff
@@ -1,44 +1,77 @@
+# TODO: Add a short module docstring above the code to:
```

Comment:
This file was also extensively re-written in #101 - I believe all docstrings are now complete so I'll remove those TODO items and leave the rest for TJ.

Reply:
Completed in PR #101